pkg/controller/storagecluster/noobaa_system_reconciler.go

```diff
@@ -51,6 +52,10 @@ func (r *ReconcileStorageCluster) ensureNoobaaSystem(sc *ocsv1.StorageCluster, r
 		if errors.IsNotFound(err) {
 
 			if cephClusterCreated {
+				// Arbitrary sleep to hopefully allow the
+				// CephCluster time to quiesce.
+				// TODO: Remove this
```

Comment:
Maybe a more elaborate TODO would be to replace this by a check that makes sure the pools are up so that we can create PVCs?